pkgs/by-name/li/librewolf-bin/package.nix

```diff
@@ -27,7 +27,7 @@ appimageTools.wrapType2 {
     description = "Fork of Firefox, focused on privacy, security and freedom (upstream AppImage release)";
     homepage = "https://librewolf.net";
     license = lib.licenses.mpl20;
-    maintainers = with lib.maintainers; [ ];
+    maintainers = with lib.maintainers; [ dwrege ];
```

Comment:
Thank you for picking this up @DominicWrege. Since you're adding yourself to this package's maintainers and not the nixpkgs maintainter-list ,I think the subject for this commit should be changed to make things more clear and avoid confusion. For example:

```diff
librewolf-bin: add dwrege as maintainer
```